src/perception/road_signs_classifier/road_signs_classifier/road_signs_classifier.py

```diff
@@ -0,0 +1,76 @@
+"""
+Package: road_signs_classifier
+File: road_signs_classifier.py
+Author: Pranav Boyapati
+"""
+
+import rclpy
+import base64
+import os
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+from navigator_msgs.msg import RoadSigns
+from navigator_msgs.msg import RoadSignsDetection
+import cv2
+from inference_sdk import InferenceHTTPClient
+
+
+class road_signs_classifier(Node):
+    def __init__(self):
+        super().__init__('road_signs_classifier')
+
+        #instantiate model client
+        self.CLIENT = InferenceHTTPClient(api_url="http://localhost:6060", api_key="BmqYjCBXZD1iPIyq09sG")
```

Comment:
Should `api_url` and `api_key` be a ROS parameter? Also, if you could add a README or documentation to your node, so people can easily run the inference server that would be great.